eaf_pdf_buffer.py

```diff
@@ -65,6 +67,8 @@ class AppBuffer(Buffer):
     def __init__(self, buffer_id, url, arguments):
         Buffer.__init__(self, buffer_id, url, arguments, False)
 
+        faulthandler.enable()
+
```

Comment:
faulthanlder only use for developer, shouldn't enable default, it will cause performance problem.